.github/workflows/kuksa_databroker_build.yml

```diff
@@ -68,6 +68,36 @@ jobs:
         working-directory: ${{github.workspace}}
         run: cargo clippy --features viss --all-targets -- -W warnings -D warnings
 
+
+  kuksa-lib:
```

Comment:
It could be discussed if this should be a job like here, or rather a workflow of its own.

Reply:
Indeed, should be fine to keep it in this workflow. No harm.